WindowsServerDocs/remote/remote-access/web-application-proxy/Publishing-Applications-with-SharePoint,-Exchange-and-RDG.md

```diff
@@ -19,6 +19,16 @@ This topic describes the tasks necessary to publish SharePoint Server, Exchange
 > [!NOTE]
 > This information is provided as-is.  Remote Desktop Services supports and recommends using [Azure App Proxy to provide secure remote access to on-premises applications](/azure/active-directory/active-directory-application-proxy-get-started).
 
+> [!IMPORTANT]
+> If you are installing the Web Application Proxy service on Windows Server 2019, you must disable HTTP2 protocol support in the WinHttp component. This is disabled by default in earlier versions of supported operating systems. Adding the following registry key and restarting the server disables it on Windows Server 2019. Note that this is a machine-wide registry key.
+>
+> ```
+> Windows Registry Editor Version 5.00
+> 
+> [HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Internet Settings\WinHttp] "EnableDefaultHttp2"=dword:00000000
+> ```
+>
```

Comment:
Cosmetic/codestyle: The current change looks pretty good to me, but we usually don't add a blank line at the end of Note blobs. Therefore, I suggest removing this single line:
```suggestion
```